### ISSUE TITLE
NoSuchMethodError in handleInitPackageResources

### ISSUE THREAD

**Issue by tonyp** [rovo89/XposedBridge#40]
I'd like to report another issue here in handleInitPackageResources().

The following code occasionally throws a NoSuchMethodError

```
XModuleResources modRes = XModuleResources.createInstance(MODULE_PATH, resparam.res);
resparam.res.setReplacement("com.android.systemui", "drawable", ...));
```

Error:

```
E/Xposed﹕ java.lang.NoSuchMethodError: No static method parsePackageLite(Ljava/lang/String;I)Landroid/content/pm/PackageParser$PackageLite; in class Landroid/content/pm/PackageParser; or its super classes (declaration of 'android.content.pm.PackageParser' appears in /system/framework/framework.jar)
```

It doesn't really negatively affect the module, the XModuleResource object gets created and can be used (with the exception of https://github.com/rovo89/XposedBridge/issues/38 in CM12).
So maybe there is something we should change in the modules code rather than in Xposed?

Complete [logcat](http://forum.xda-developers.com/xposed/problem-server-classes-resource-hooks-t3044270/post59584611#post59584611)

I've seen this error as well for the new Lollipop-compatible AppSettings version from rovo, error log:
http://pastebin.com/mJbYzv5H

In the past I've seen the same error in the Xposed Log for Gravitybox, too - but right now it isn't showing so I either changed some settings or @C3C0 fixed it / catched it?

**Comment by pylerSM** [comment]
This maybe fixes it
https://github.com/rovo89/XposedBridge/pull/34

**Comment by rovo89** [comment]
Fixed by #34.
